src/cloudWatchLogs/registry/logStreamRegistry.ts

```diff
@@ -198,37 +179,72 @@ export class LogStreamRegistry {
         return (log && log.busy) ?? false
     }
 
-    private setLog(uri: vscode.Uri, stream: CloudWatchLogStreamData): void {
+    private setLog(uri: vscode.Uri, stream: CloudWatchLogsData): void {
         this.activeStreams.set(uri.path, stream)
     }
 
-    private getLog(uri: vscode.Uri): CloudWatchLogStreamData | undefined {
+    private getLog(uri: vscode.Uri): CloudWatchLogsData | undefined {
         return this.activeStreams.get(uri.path)
     }
+}
 
-    private async getLogEventsFromUriComponents(
-        logGroupInfo: {
-            groupName: string
-            streamName: string
-            regionName: string
-        },
-        nextToken?: string
-    ): Promise<CloudWatchLogs.GetLogEventsResponse> {
-        const client: CloudWatchLogsClient = globals.toolkitClientBuilder.createCloudWatchLogsClient(
-            logGroupInfo.regionName
+export async function getLogEventsFromUriComponents(
```

Comment:
is the name `getLogEventsFromUriComponents` still relevant? it's almost confusing because it's overspecified. Maybe just `getLogEvents` works.